src/nsls2api/cli/cli.py

```diff
@@ -132,10 +132,9 @@ def main(
     """
     if version:
         try:
-            from nsls2api._version import version as ver
-
-            console.print(f"[info]NSLS-II API CLI version: {ver}")
-        except ImportError:
+            version = get_version()
+            console.print(f"[info]NSLS-II API CLI version: {version}")
+        except importlib.metadata.PackageNotFoundError:
             console.print("[warning]Version information not available")
         raise typer.Exit()
```

Comment:
This try/except block is probably no longer necessary.
```suggest
         version = get_version()
         console.print(f"[info]NSLS-II API CLI version: {version}")
```

Reply:
🤔 I don't see this code in my branch - am i doing something stupid ?

Reply:
Indeed - I was just automatically adding error handling when it's not needed - thank you for the catch.